example/src/main/java/org/blockstack/android/MainActivity.kt

```diff
@@ -283,13 +271,8 @@ class MainActivity : AppCompatActivity() {
         }
 
         if (intent?.action == Intent.ACTION_VIEW) {
-            lifecycleScope.launch(Dispatchers.Main) {
-                val response = BlockstackConnect.handleAuthResponse(intent)
-                if (response.value != null) {
-                    runOnUiThread {
-                        onSignIn(response.value!!)
```

Comment:
Why do you not call `onSignIn` anymore?

Reply:
We are now calling handleAuthResponse, that calls onSignIn.
This because on the example app we have both BlockstackConnect and the "Browser" one.
This method (BlockstackConnect.handleAuthResponse) was design only to connect and I noticed a few issues :)